README.md

```diff
@@ -1,27 +1,24 @@
 # Virtual Pet
 
-**Real pets are great, but they do have drawbacks. They smell. Your mum might be allergic to them. Maybe you just aren't allowed one in your flat.**
+## The repository contains test cases for a pet.js file that constructs a *Virtual Pet object*, that has numerous properties, including methods attributed to prototypes.
+-----
```

Comment:
This is a nice basic README, but it is lacking in a few things. Some things that would make it much better:
- what technologies are used,
- installation instructions,
- how does the app work and what does the user do?
- What would you do if you had more time?
- your name!

For the formatting, heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)